doctest/test_mapping/nested_objects.json

```diff
@@ -0,0 +1,47 @@
+{
+  "mappings": {
+    "properties": {
+      "message": {
+        "type": "nested",
+        "properties": {
+          "info": {
+            "type": "keyword",
+            "index": "true"
+          },
+          "author": {
+            "type": "keyword",
+            "fields": {
```

Comment:
It is possible to have `keyword` with `fields`? How it works?

Reply:
Not entirely sure, this was taken from [nested_type_index_mapping.json](https://github.com/opensearch-project/sql/blob/main/integ-test/src/test/resources/indexDefinitions/nested_type_index_mapping.json)